### ISSUE TITLE
Criar classe Caixa Registradora

### ISSUE THREAD

**Issue by luizlaljr** [crismotinha/supermercado-coop#6]
A caixa deve receber os produtos e suas quantidades uma a uma e depois trazer o valor da compra.
Depois de ser efetuado o pagamento na modalidades propostas no trabalho, deve ser realizada a baixa do estoque.

**Comment by luizlaljr** [comment]
Como sugestão, a caixa registradora deve ter um atributo estoque que deverá ser passado no seu construtor.

**Comment by VonLisboa** [comment]
Não sei se entendi mas Caixa faria o papel da classe Pedido, que criei anteriormente.
Seria isso? @luizlaljr @crismotinha

**Comment by luizlaljr** [comment]
Sim, só atualizei o nome para o informado na proposta do trabalho para manter o mesmo padrão.
